src/utilities/convex_hull.zig

```diff
@@ -721,19 +820,179 @@ test "Merge convex hull with single element's convex hull" {
     defer convex_hull_one.deinit();
 
     for (0..num_points) |i| {
-        try convex_hull_one.addPoint(.{ .time = i, .value = rnd.random().float(f64) });
+        try convex_hull_one.add(.{ .time = i, .value = rnd.random().float(f64) });
     }
 
     var convex_hull_two = try ConvexHull.init(allocator);
     defer convex_hull_two.deinit();
 
-    try convex_hull_two.addPoint(.{ .time = num_points, .value = rnd.random().float(f64) });
+    try convex_hull_two.add(.{ .time = num_points, .value = rnd.random().float(f64) });
 
-    try convex_hull_one.merge(&convex_hull_two);
+    try convex_hull_one.merge(&convex_hull_two, null);
 
     try testConvexHullProperty(&convex_hull_one);
 }
 
+test "Merge not-in-place convex hull with single element's convex hull" {
+    const num_points: usize = 100;
```

Comment:
Seems `num_points` use the same value in the tests, maybe see if it can be made a private shared constant so the amount of test data to generate is simpler to change for all tests.

Reply:
I fixed this by adding a local variable that takes the number_of_values from `src/tester.zig`. However, in one particular test the number of points has to be 100 because the test is tailored for that amount. So, if the variable changes in `src/tester.zig` this test will not work. The test is 
test "Merge not-in-place does not modify the convex hulls one and two".
Should we have a `local_num_points` for this particular test?

Reply:
> I fixed this by adding a local variable that takes the number_of_values from `src/tester.zig`. 
Sounds good, although unclear why the constant from `src/tester.zig` is not used directly instead of assigning the value to a local variable.

> However, in one particular test the number of points has to be 100 because the test is tailored for that amount. So, if the variable changes in `src/tester.zig` this test will not work. The test is test "Merge not-in-place does not modify the convex hulls one and two". Should we have a `local_num_points` for this particular test?
If the test is purposely made for 100 data points I think having a local variable with the amount makes sense. Although, probably a good idea to add a comment above the variable to make it clear why the value from `src/tester.zig` is not used.

Reply:
Good news, trying to understand why the test was failing when the number of points was different from 100 I found a bug. Now is fixed and the number of elements can take any value >= 2. I added a comment in that variable explaining that.